markdown/en/ff-loading-spinner.md

```diff
@@ -0,0 +1,74 @@
+## Overview
+
+`ff-loading-spinner` is a leightweight spinning loading indicator that is 
+triggered by common events fired by FACT-Finder Web Components. It uses 
+inexpensive CSS-Animations and does not trigger `layout` or `paint` 
```

Comment:
[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations) spells `CSS-Animations` as `CSS animations`. Let's pick their spelling.

Reply:
I prefer the hyphenated spelling, however we have similar terms where we aren't using a consistent spelling:
- HTML-Element vs. HTML element vs. HTML Element
- XHR vs. xhr vs. XHR Request vs. XHR request (the request is redundant)
We actually tried fixing this in #109 

I will use the MDN's spelling for now. I think we should create a spelling guide or a lexicon (in confluence maybe?).

Reply:
Actually, I was working on a parsing tool for the documentation which you can feed a dictionary with terms to check and then it will list all inconsistencies for each term.
This might actually be worth looking further into.
A spelling guide has to be read and implemented by humans who are clearly the source of error here 😄 
I have a working prototype. If you're interested, I can introduce you two.